src/org/zaproxy/zap/extension/saml/ui/SamlManualEditor.java

```diff
@@ -223,7 +223,7 @@ public void focusLost(FocusEvent e) {
      * @param msg
      */
     private void updateResponse(HttpMessage msg) {
-        respBodyTextPane.setText(msg.getResponseBody().createCachedString("UTF-8"));
+        respBodyTextPane.setText(msg.getResponseBody().toString());
```

Comment:
The change was already applied over here: https://github.com/zaproxy/zap-extensions/pull/445

It seems like a merged change on this PR, when you probably meant for it to be a rebased change....or perhaps you applied the change manuallyour and then it was processed via the pull linked above?